### ISSUE TITLE
Compact does not include exception

### ISSUE THREAD

**Issue by JDrizzy** [julienXX/resque-slack#8]
Firstly, thank you for this gem and your work! 

Testing verbose level `:compact` I discovered that the exception description is meant to be displayed in the slack message, unfortunately this is not the case. To test this I used the following script:

```Ruby
require 'resque/failure/slack'
Resque::Failure::Slack.configure do |config|
  config.channel = ''
  config.token = ''
  config.level = :compact
end
resque_slack = Resque::Failure::Slack.new(
  ArgumentError,
  'JobName',
  'high',
  {
    data: []
  }
)
resque_slack.text

# Returns
# "JobName failed processing high\nPayload:\n  {:data=>[]}\n\n"
```

#### Proposed Changes

Since `msg_exception` last line returns `nil` for level `:compact`, I suggest that `return str` is inserted at the end of the function to ensure both levels `:verbose` and `:compact` work as expected:

Before:
```Ruby
def msg_exception(backtrace)
  str = "Exception:\n#{exception}"
  str += "\n#{exception.backtrace.map { |l| '  ' + l }.join('\n')}" if backtrace
end
```

After
```Ruby
def msg_exception(backtrace)
  str = "Exception:\n#{exception}"
  str += "\n#{exception.backtrace.map { |l| '  ' + l }.join('\n')}" if backtrace
  return str
end
```

Re-running the script now returns the following:
```Ruby
# "JobName failed processing high\nPayload:\n  {:data=>[]}\nException:\nArgumentError\n"
```

Please let me know if you would like me to raise PR for these changes.

```
ruby - 2.5.1
rails - 5.2.2
resque - 1.27
```

**Comment by julienXX** [comment]
Hello @JDrizzy thanks a lot for pointing that out, I'll happily review a PR if you want to submit one :+1:

**Comment by JDrizzy** [comment]
@julienXX thanks for checking out the above.

I just investigated the current repository changes and noticed that class `Notification` was added that will resolve the issue raised. I was unfortunately looking at/testing an old version of the changes in gem 0.1.0 locally (smart move). 

I will close this issue for now with the hope that a new version release will be built with the latest changes.

Thanks again.
